prog/install_rhizome.rb

```diff
@@ -10,7 +10,10 @@ class Prog::InstallRhizome < Prog::Base
     tar = StringIO.new
     Gem::Package::TarWriter.new(tar) do |writer|
       base = Config.root + "/rhizome"
-      Dir.glob(["Gemfile", "Gemfile.lock", "common/**/*", "#{frame["target_folder"]}/**/*"], base: base).map do |file|
+      target_folder = frame["target_folder"]
+      files = Dir.glob(["Gemfile", "Gemfile.lock", "common/**/*", "#{target_folder}/**/*"], base: base)
+      files -= Dir.glob("#{target_folder}/**/*_spec.rb", base: base) unless frame["include_spec"]
+      files.map do |file|
```

Comment:
The multiple traversals here are slightly unsatisfying. Perhaps do away with the `map` and this set-subtraction from two traversals and use `Find` and `fnmatch`, kinda like this:

```
files = Find.find("rhizome").lazy
if frame["include_spec"]
  files
else
  files.filter_map { File.fnmatch("**/spec", _1) ? Find.prune : _1 }
end
```

Note that `lazy` increases your composition options when chaining in enumerable methods, like `filter_map`, without materializing intermediate results necessarily.

Reply:
getting an elegant solution of this, I'm playing with it, not straightforward, please hold...

Reply:
Why do we use `Dir.glob(...).map` in the first place (even before this commit)?

How about
```ruby
Dir.glob(...).each do |file|
  next if !frame["include_spec"] && file.end_with?("_spec.rb")
  #... put to tar...
end
```
?